internal/collector/common/path.go

```diff
@@ -1,26 +1,25 @@
 package common
 
 import (
-	"fmt"
 	"path/filepath"
 )
 
 // SecretPath is the path for any secret visible to the collector
 func SecretPath(secretName string, file string) string {
-	return fmt.Sprintf("%q", filepath.Join("/var/run/ocp-collector/secrets", secretName, file))
+	return filepath.Join("/var/run/ocp-collector/secrets", secretName, file)
```

Comment:
Are these used in config gen and for mounting volume paths?  I think one place needs quotes and the other does not which means we'll need changes to accommodate both